### DISCUSSION TITLE
A Melhor Cena de Ciência de Todas

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#464] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/a-melhor-cena-de-ciencia-de-todas/

**Comment by one-eyed-willy** [comment]
Foi assim que eu quase matei 40 alunos no ensino médio numa das poucas aulas de química em que fomos pro laboratório :pensive:

**Comment by tali-zorah** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/videos/a-melhor-cena-de-ciencia-de-todas/#comment_1129013096>):

> Foi assim que eu quase matei 40 alunos no ensino médio numa das poucas aulas de química em que fomos pro laboratório 😔

*Keelah!*

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/videos/a-melhor-cena-de-ciencia-de-todas/#comment_1129013096>):

> Foi assim que eu quase matei 40 alunos no ensino médio numa das poucas aulas de química em que fomos pro laboratório 😔

![null](<https://i.postimg.cc/BnRTN324/gugure-kokkuri-san-hello-phone-police.png>)

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/videos/a-melhor-cena-de-ciencia-de-todas/#comment_1129128185>):

> [![null](<https://camo.githubusercontent.com/c98be41ec9bd98d7c1a65a17d91fd5b62bbbe16d82c59b256d124ee5079d86f2/68747470733a2f2f692e706f7374696d672e63632f426e52544e3332342f6775677572652d6b6f6b6b7572692d73616e2d68656c6c6f2d70686f6e652d706f6c6963652e706e67>)](<https://camo.githubusercontent.com/c98be41ec9bd98d7c1a65a17d91fd5b62bbbe16d82c59b256d124ee5079d86f2/68747470733a2f2f692e706f7374696d672e63632f426e52544e3332342f6775677572652d6b6f6b6b7572692d73616e2d68656c6c6f2d70686f6e652d706f6c6963652e706e67>)

Eu sempre gostei de química, mas sempre fui péssimo na matéria. Daí eu ~~trafiquei~~ levei um pouco de açúcar pra escola pra fazer a experiência do "monstro de açúcar", uma gosma preta que sai do copo quando em contato com ácido sulfúrico. Pra mim já tava bom ali, mas o professor quis deixar "mais empolgante" e mandou pra colocar um pouquinho de... ácido fluorídrico, eu acho, não lembro.

Só que eu não ouvi o "pouquinho" e despejei, tipo, uns 5ml. Subiu uma fumaceira amarela fedida, tipo aquelas de estádio de futebol, e todo mundo ficou tossindo com força o resto do período todo. Na sala ele deu uma p\*ta bronca em todo mundo, mas ficou só olhando na minha cara, e explicou que aquela fumaça era óxido nitroso, que era perigoso e tal.
